engine/src/engine.ts

```diff
@@ -191,14 +191,18 @@ export default class Engine {
     }
 
     const move = _move.trim();
+    let moveToShow = move;
+    if (move.split(" ")[1] == Command.Pass) {
```

Comment:
use `move.startsWith(Command.Pass)` instead

Reply:
the move starts with the faction, but `move.includes(" " + Command.Pass + " ")` works